docs/guides/cloud/github-integration.mdx

```diff
@@ -375,6 +375,17 @@ integration settings:
   alt="Status checks settings"
 />
 
+:::tip
+
+<strong>
```

Comment:
I missed this in the github guide updates so adding it here. @pstakoun like the gitlab one, let me know if you want any changes to this CTA.